src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt

```diff
@@ -0,0 +1,51 @@
+package no.nav.syfo.client.altinn
+
+import no.altinn.schemas.services.intermediary.receipt._2009._10.ReceiptStatusEnum
+import no.altinn.schemas.services.serviceengine.correspondence._2010._10.InsertCorrespondenceV2
+import no.nav.syfo.domain.Virksomhetsnummer
+import org.slf4j.LoggerFactory
+import java.util.*
+
+class AltinnClient(
+    altinnWsUrl: String,
+    private val username: String,
+    private val password: String
+) {
+    private val SYSTEM_USER_CODE = "NAV_DIGISYFO"
+    private val log = LoggerFactory.getLogger(AltinnClient::class.java)
+    private val iCorrespondenceAgencyExternalBasic = createPort(altinnWsUrl)
+
+    fun sendToVirksomhet(
+        brevId: UUID,
+        brevPdf: ByteArray,
+        virksomhetsnummer: Virksomhetsnummer,
+    ) {
+        val virksomhetsBrevAltinnWS = AltinnMapper.brevTilWSCorrespondence(brevId, brevPdf, virksomhetsnummer)
+        sendMelding(virksomhetsBrevAltinnWS, brevId)
```

Comment:
Bruke named parameters i disse to funksjonskallene?

Reply:
funksjon💀💀